packages/account/src/Configs/personal-details-config.ts

```diff
@@ -38,9 +36,9 @@ export const personal_details_config = ({
     const min_phone_number = 9;
     const max_phone_number = 35;
 
-    const default_residence = real_account_signup_target === 'maltainvest' ? account_settings?.residence : '';
+    const default_residence = real_account_signup_target === 'maltainvest' ? account_settings?.residence ?? '' : '';
```

Comment:
It's a ternary condition already, do we really need this fallback?

Reply:
I think it is done to prevent default_residence from taking undefined incase `residence` is not present in account_settings

Reply:
It’s been changed a little bit. hope now it looks like better